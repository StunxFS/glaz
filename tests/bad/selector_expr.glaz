fn x() (i32, bool) {
  return (10*10, true);
}

fn main() {
  let a = x();
  _ = a.s3;
}
