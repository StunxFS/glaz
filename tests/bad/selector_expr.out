tests/bad/selector_expr.glaz:7:10: error: type `(i32, bool)` has no field `s3`
     | 
   7 |   _ = a.s3;
     |         ^^
     = help: 2 possibilities: `s1`, `s2`
tests/bad/selector_expr.glaz:7:8: error: invalid use of void expression
     | 
   7 |   _ = a.s3;
     |       ^^^^
glaz error: could not compile `selector_expr`, aborting due to 2 previous errors.
